native/c/zbase64.h

```diff
@@ -119,15 +109,44 @@ inline size_t max_decoded_size(size_t input_size)
   return (input_size / 4) * 3;
 }
 
+inline size_t load_leftover(const char *input, size_t input_len, std::vector<char> &left_over, std::vector<char> &temp_storage)
+{
+  // Handle leftover data combination
+  size_t total_size = left_over.size() + input_len;
+  temp_storage.resize(total_size);
+
+  std::memcpy(&temp_storage[0], &left_over[0], left_over.size());
+  std::memcpy(&temp_storage[0] + left_over.size(), input, input_len);
```

Comment:
Suggestion, not requesting changes: If `std::vector::data()` is available in our compiler, I'd recommend using that instead of `&vec[0]` ^^ otherwise feel free to keep as-is

Reply:
I would like to use `std::vector::data()` but `xlc` doesn't support it 😿

Reply:
I'm not surprised that it doesn't support it, just disappointed 😄 
Thanks for testing!